### ISSUE TITLE
The MeasuredDimensions type is equal to the any type

### ISSUE THREAD

**Issue by jobpaardekooper** [software-mansion/react-native-reanimated#4556]
### Description

The `MeasuredDimensions` type is `any`. This worked fine in reanimated 2 but has been broken for me since reanimated 3.

<img width="342" alt="Screenshot 2023-06-11 at 10 30 47" src="https://github.com/software-mansion/react-native-reanimated/assets/34612663/3eb7ecd5-585d-4f2f-9e22-bc3ea71cbeae">

It is defined [in the docs](https://docs.swmansion.com/react-native-reanimated/docs/api/nativeMethods/measure/#returns) as having some properties (so it should not be `any` type).

The other typings are working perfectly fine for me.

### Steps to reproduce

1. Install the latest version of react-native-reanimated
2. Add the following code your project:
```ts
import { MeasuredDimensions } from "react-native-reanimated"
```
You will observe that `MeasuredDimensions` is equal to the `any` type.

### Snack or a link to a repository

Not needed

### Reanimated version

3.2.0

### React Native version

0.71.8

### Platforms

Android, iOS

### JavaScript runtime

None

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Acknowledgements

Yes

**Comment by tomekzaw** [comment]
Hey @jobpaardekooper, thanks for reporting this problem. Indeed, `MeasuredDimensions` resolves to `any`. The root cause of this issue might be invalid paths in `react-native-reanimated.d.ts`. I believe this problem (as well as many others) will be successfully fixed thanks to @tjzel's efforts on fixing our TypeScript types and configs (see https://github.com/software-mansion/react-native-reanimated/pull/4519).

**Comment by jobpaardekooper** [comment]
Thanks for the quick reply! I will leave this open for others to find until #4519 is merged.

**Comment by claudiorivera** [comment]
I was getting this when I ran `npx expo install react-native-reanimated`, which resolved to `~3.3.0`.

After manually upgrading to `^3.4.2`, this issue is now gone 🎉

**Comment by jobpaardekooper** [comment]
This has indeed also been fixed for me. I will close the issue. Thanks for the great work. 🚀
